fuzz/fuzz_helper/src/lib.rs

```diff
@@ -189,6 +189,8 @@ pub fn process_ctap_structured(data: &[u8], input_type: InputType) -> arbitrary:
         }
     };
 
+    env.seed_rng_from_u64(u64::arbitrary(unstructured)?);
```

Comment:
I would move this at the very top to be the first usage of `unstructured`. This would make the "fuzzing format" more stable because it would be 64 bits prefix followed by command input (which is variable sized).